services/storage/client-sdk/python/simcore_service_storage_sdk/rest.py

```diff
@@ -14,6 +14,7 @@
 import io
 import json
```

Comment:
FYI: this part is auto-generated ...  
in any case, I am sure we will have to get rid of it

Reply:
I am aware of this, I've put a test in place to make sure people do not regenerate the client and forget the timeouts.